plugins/hypervisors/kvm/src/test/java/com/cloud/hypervisor/kvm/resource/LibvirtVMDefTest.java

```diff
@@ -238,6 +238,39 @@ public void testDiskDefWithEncryption() {
         assertEquals(disk.toString(), expectedXML);
     }
 
+    @Test
+    public void testDiskDefWithMultipleHosts() {
+        String path = "/mnt/primary1";
+        String host = "10.11.12.13,10.11.12.14,10.11.12.15";
```

Comment:
Could you also make a test-case with IPv6 addresses? Just to make sure we cover that as well

Reply:
@wido 
It seems monitor with ipv6 address is not suported in this PR.

what is currently supported and any issue in following scenarios ?
- ipv6 addr without port
- ipv6 addr with port
I do not have a ceph env with ipv6 address , it is difficult to verify.

Reply:
@wido 
please ignore my previous comment
I will use ULA addresses in testing

Reply:
> @wido please ignore my previous comment I will use ULA addresses in testing

You can also use the 2001:db8::/32 range for this: https://www.rfc-editor.org/rfc/rfc3849

That would be sufficient to write your test-cases with.